docs/tracking.md

```diff
@@ -1 +1,1058 @@
+$$
+
+where the linear function $\mathbf F_{k-1}$ transports the state vector at
+step $k-1$ to step $k$. $\vec w_{k-1}$ is additional so-called process noise
+that affects the transport additively. Each step has an associated
+measurement, with the fixed relationship between the measurement and the state vector
+
+$$
+  \vec m_k = \mathbf H_k \vec x_k + \epsilon_k.
+$$
+
+Here, $\mathbf H_k$ is the *measurement mapping function*, which
+transforms the state vector into the measurement space. In the ideal case,
+this purpose can be achieved by a simple projection matrix, which extracts a
+subspace of the state vector. Additionally, $\epsilon_k$ represents the
+measurement uncertainty.
+
+The Kalman fit process is divided into different phases:
+
+1. **Prediction** of the state vector at the next step $k+1$ based on the information at the current step $k$.
+2. **Filtering** of the prediction by incorporating the measurement associated to the step
+3. **Smoothing** of the state vector by walking back the steps and using information for the subsequent step $k+1$ to improve the estimate at the current step $k$.
+
+An illustration of these concepts is found in {numref}`kalman_filter`. Here,
+a series of three sensors is shown with measurements on them. The KF
+then predicts the track parameters at an intersection, shown in blue.
+Subsequently, a filtered set of parameters is calculated as a mixture between
+the measurement and the prediction. Not shown in this picture is the
+smoothing step.
+
+(kalman_filter)=
+
+:::{figure} /figures/tracking/kalman.svg
+:width: 400px
+:align: center
+
+Illustration of the KF. Two of the three stages, the prediction and the
+filtering are shown. The filtering updates the prediction with information from
+the measurement.
+:::
+
+
+In many cases, the first two phases run in tandem, with prediction and
+filtering happening alternatingly at each step. The smoothing phase is
+launched once the last measurement has been encountered.
+Starting from a state $k$, first, a prediction of the state vector at the
+next measurement location is obtained via
+
+(kf_pred)=
+$$
+  \vec x_k^{k-1} = \mathbf F_{k-1} \vec x_{k-1},
+$$
+
+with the linear transport function from above. $\vec x_k^{k-1}$ is
+the prediction of the state vector at step $k$ based on step $k-1$. The next
+stage is the filtering. Here, the state vector is refined by taking into
+account the measurement at the current step. Following one of two variants of
+filtering from [^Fruhwirth:1987fm], the gain matrix formalism, the state
+vector is updated like
+
+$$
+  \vec x_k = \vec x_k^{k-1} + \mathbf K_k \left( \vec m_k - \mathbf H_k \vec x_k^{k-1} \right),
+$$
+
+with the *Kalman gain matrix*
+
+$$
+  \mathbf K_k = \mathbf C_k^{k-1} \mathbf H_k^\mathrm{T}
+    \left(
+      \mathbf V_k + \mathbf H_k \mathbf C_k^{k-1} \mathbf H_k^\mathrm{T}
+    \right)^{-1}
+    %= \mathbf C_k \mathbf H_k^\mathrm{T} \mathbf G_k
```

Comment:
Was it in your intention to comment this line out?
It also happens below in some places.

Reply:
I don't remember, this was for my thesis. Probably fine to remove the comments.
```suggestion
```